File: TP3_Python/Codingan/Internal.py
class Internal:
    def __init__(self, nama="", spesifikasi="", merk_internal=""):
        self.nama = nama
        self.spesifikasi = spesifikasi
        self.merk_internal = merk_internal

    def set_nama(self, nama):
        self.nama = nama

    def set_spesifikasi(self, spesifikasi):
        self.spesifikasi = spesifikasi

    def set_merk_internal(self, merk_internal):
        self.merk_internal = merk_internal

    def get_merk_internal(self):
        return self.merk_internal

    def get_spesifikasi(self):
        return self.spesifikasi

    def get_nama(self):
        return self.nama

File: TP3_CPP/Codingan/Internal.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Internal
{
private:
    string Nama;
    string Spesifikasi;
    string MerkINTERNAL;

public:
    Internal()
    {
    }

    Internal(string Nama, string Spesifikasi, string MerkINTERNAL)
    {
        this->MerkINTERNAL = MerkINTERNAL;
        this->Spesifikasi = Spesifikasi;
        this->Nama = Nama;
    }

    // Setter
    void setNama(string Nama)
    {
        this->Nama = Nama;
    }

    void setSpesifikasi(string Spesifikasi)
    {
        this->Spesifikasi = Spesifikasi;
    }

    void setMerkINTERNAL(string MerkINTERNAL)
    {
        this->MerkINTERNAL = MerkINTERNAL;
    }

    // Getter
    string getMerkINTERNAL()
    {
        return MerkINTERNAL;
    }

    string getSpesifikasi()
    {
        return Spesifikasi;
    }

    string getNama()
    {
        return Nama;
    }

    ~Internal()
    {
    }
};
